src/Components/FormCreation/CardComponents/FileQuestion.tsx

```diff
@@ -60,68 +80,92 @@ function FileQuestion({
     null,
     []
   );
-  let link = "";
-  //Create a link to download the file :)
-  if (!loadFile.isPending) {
-    const bytes = new Uint8Array(loadFile.value.Body.data); // pass your byte response to this constructor
-    const blob = new Blob([bytes], { type: "application/octet-stream" }); // change resultByte to bytes
-    link = window.URL.createObjectURL(blob);
 
-    // const headerImg = document?.querySelector<HTMLImageElement>("#image");
-    // // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
-    // headerImg!.src = link;
+  // let link = "";
+  // //Create a link to download the file :)
+  // if (!loadFile.isPending && loadFile.value && loadFile.value.Body) {
+  //   const bytes = new Uint8Array(loadFile.value.Body.data); // pass your byte response to this constructor
+  //   const blob = new Blob([bytes], { type: "application/octet-stream" }); // change resultByte to bytes
+  //   link = window.URL.createObjectURL(blob);
+
+  //   // const headerImg = document?.querySelector<HTMLImageElement>("#image");
+  //   // // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+  //   // headerImg!.src = link;
+  // }
```

Comment:
Is there a future use case for this or should we delete it?

Reply:
Yes. When displaying on decision canvas we will need this code. Whether we purpose this component as a read-only version or move it somewhere else